src/libraries/System.Runtime/tests/System/DateTimeOffsetTests.cs

```diff
@@ -1425,5 +1425,17 @@ public static void UnixEpoch()
         {
             VerifyDateTimeOffset(DateTimeOffset.UnixEpoch, 1970, 1, 1, 0, 0, 0, 0, 0, TimeSpan.Zero);
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Android | TestPlatforms.iOS | TestPlatforms.tvOS)]
+        public static void UtcIsNotLocalTime()
+        {
+            DateTimeOffset utcNow = DateTimeOffset.UtcNow;
+            
+            string utcTime = utcNow.ToString();
+            string localTime = utcNow.ToLocalTime().ToString();
+
+            Assert.NotEqual(utcTime, localTime);
```

Comment:
Is the reason this test is valid because a UTC DateTimeOffset will append a `Z` to the end, where a local DateTimeOffset, even if it is in the same time zone as UTC will print `+0` (or similar)?

Reply:
I think they'll print the same thing if they are in the same timezone as UTC. The test assumes the devices aren't.

Reply:
Is that a valid assumption?

Reply:
I think so.  Without the patch, the test fails.  Do you know of anything else to try?

Reply:
What about:

```C#
 Assert.NotEqual(TimeZoneInfo.Utc.StandardName, TimeZoneInfo.Local.StandardName);
```

?

Reply:
Thanks!  I like that one better.